libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -32,27 +35,30 @@
   *ngIf="metadata.lineage"
   [title]="'record.metadata.origin' | translate"
 >
-  <p class="mb-5 pt-4 whitespace-pre-line break-words" gnUiLinkify>
+  <p
+    class="text-gray-900 mb-5 pt-4 whitespace-pre-line break-words"
+    gnUiLinkify
+  >
     {{ metadata.lineage }}
   </p>
   <div
     class="py-4 px-6 rounded bg-gray-100 grid grid-cols-2 gap-y-[10px] gap-x-[20px] text-gray-700 info-grid"
   >
     <div *ngIf="metadata.recordUpdated">
       <p class="text-sm" translate>record.metadata.updatedOn</p>
-      <p class="text-primary font-medium mt-2">
+      <p class="text-primary font-medium mt-[4px]">
```

Comment:
`mt-1` should be the same I think

Reply:
I think it is more explixit to put it in px, because it reflects what is in the mockup. In terms of generated css it's almost the same.

Reply:
Ok, so far we've privileged the non interpolated values (also for the lightest CSS possible, I think). I don't have a strong opinion on this, but maybe we should agree on a best practice.